Test failure: `com.linecorp.centraldogma.server.HealthCheckServiceTest.healthCheck()`

```
com.linecorp.armeria.client.UnprocessedRequestException: io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: /127.0.0.1:44667
	at app//com.linecorp.armeria.client.UnprocessedRequestException.of(UnprocessedRequestException.java:45)
	at app//com.linecorp.armeria.client.HttpChannelPool.notifyConnect(HttpChannelPool.java:553)
	at app//com.linecorp.armeria.client.HttpChannelPool.lambda$connect$2(HttpChannelPool.java:375)
	at app//io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:603)
	at app//io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:570)
	at app//io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:505)
	at app//io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:649)
	at app//io.netty.util.concurrent.DefaultPromise.setFailure0(DefaultPromise.java:642)
	at app//io.netty.util.concurrent.DefaultPromise.tryFailure(DefaultPromise.java:131)
	at app//com.linecorp.armeria.client.HttpChannelPool.lambda$connect$3(HttpChannelPool.java:426)
	at app//io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:603)
	at app//io.netty.util.concurrent.DefaultPromise.notifyListeners0(DefaultPromise.java:596)
	at app//io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:572)
	at app//io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:505)
	at app//io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:649)
	at app//io.netty.util.concurrent.DefaultPromise.setFailure0(DefaultPromise.java:642)
	at app//io.netty.util.concurrent.DefaultPromise.tryFailure(DefaultPromise.java:131)
	at app//io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.fulfillConnectPromise(AbstractEpollChannel.java:728)
	at app//io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:747)
	at app//io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:611)
	at app//io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.handle(AbstractEpollChannel.java:472)
	at app//io.netty.channel.epoll.EpollIoHandler$DefaultEpollIoRegistration.handle(EpollIoHandler.java:307)
	at app//io.netty.channel.epoll.EpollIoHandler.processReady(EpollIoHandler.java:489)
	at app//io.netty.channel.epoll.EpollIoHandler.run(EpollIoHandler.java:444)
	at app//io.netty.channel.SingleThreadIoEventLoop.runIo(SingleThreadIoEventLoop.java:207)
	at app//io.netty.channel.SingleThreadIoEventLoop.run(SingleThreadIoEventLoop.java:178)
	at app//io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:1073)
	at app//io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at app//io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base@17.0.16/java.lang.Thread.run(Thread.java:840)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: /127.0.0.1:44667
Caused by: java.net.ConnectException: finishConnect(..) failed: Connection refused
	at io.netty.channel.unix.Errors.newConnectException0(Errors.java:166)
	at io.netty.channel.unix.Errors.handleConnectErrno(Errors.java:131)
	at io.netty.channel.unix.Socket.finishConnect(Socket.java:360)
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.doFinishConnect(AbstractEpollChannel.java:764)
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:741)
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:611)
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.handle(AbstractEpollChannel.java:472)
	at io.netty.channel.epoll.EpollIoHandler$DefaultEpollIoRegistration.handle(EpollIoHandler.java:307)
	at io.netty.channel.epoll.EpollIoHandler.processReady(EpollIoHandler.java:489)
	at io.netty.channel.epoll.EpollIoHandler.run(EpollIoHandler.java:444)
	at io.netty.channel.SingleThreadIoEventLoop.runIo(SingleThreadIoEventLoop.java:207)
	at io.netty.channel.SingleThreadIoEventLoop.run(SingleThreadIoEventLoop.java:178)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:1073)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:840)

```

--- a/server/src/test/java/com/linecorp/centraldogma/server/HealthCheckServiceTest.java
+++ b/server/src/test/java/com/linecorp/centraldogma/server/HealthCheckServiceTest.java
@@ -45,6 +45,7 @@ class HealthCheckServiceTest {
 
         try (CentralDogma dogma = new CentralDogmaBuilder(rootDir.toFile())
                 .port(randomPort, SessionProtocol.HTTP)
+                .gracefulShutdownTimeout(new GracefulShutdownTimeout(1000, 1000))
                 .build()) {
             dogma.start().join();
             final ServerPort serverPort = dogma.activePort();

